components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -62,7 +62,14 @@ def main(argv=None):
   logging.info('Submitting Batch Transformation request to SageMaker...')
   batch_job_name = _utils.create_transform_job(client, vars(args))
   logging.info('Batch Job request submitted. Waiting for completion...')
-  _utils.wait_for_transform_job(client, batch_job_name)
+
+  try:
+    _utils.wait_for_transform_job(client, batch_job_name)
+  except:
```

Comment:
whats the need for an empty catch block? can we remove this?

generally should avoid having a generic catch block which does not catch a specific exception

Reply:
Hey Suraj this PR is already merged 

we need that to print cw logs even if the job fails 
it'n not empty. It raises the exception so as to keep the error stack trace intact

Reply:
The code block below will throw exception as well
```
try:
  raise ValueError()
finally:
  print("hi")
```
can you provide more details on how re-raising keeping the stack trace intact ? (I am missing something here)

I understand PR is merged, we can do separate PR if needed